src/nginxconfig/generators/conf/nginx.conf.js

```diff
@@ -51,13 +52,16 @@ export default (domains, global) => {
     // HTTP (kv so we can use the same key multiple times)
     config.http = [];
 
-    if (global.php.phpBackupServer.computed)
-        config.http.push(['upstream php', {
-            server: [
-                phpPath(global),
-                `${phpPath(global, true)} backup`,
-            ],
-        }]);
+    domains.forEach(domain => {
+        if (domain.php.phpBackupServer.computed){
+            config.http.push([`upstream ${phpUpstream(domain)}`, {
+                server: [
+                    phpPath(domain),
+                    `${phpPath(domain, true)} backup`,
+                ],
+            }]);
+        }
+    });
```

Comment:
Instead of defining all the upstream configs here, I think it'd make sense to move this generation to the top of each website config (outside the server blocks).

Reply:
This can be added here?
https://github.com/digitalocean/nginxconfig.io/blob/4cfdcdb8d1c2df185e9ba437ca9c79e7c2ef17e8/src/nginxconfig/generators/conf/website.conf.js#L286

Reply:
Here, it needs to be in the `http` context (the `config` variable:

https://github.com/digitalocean/nginxconfig.io/blob/4cfdcdb8d1c2df185e9ba437ca9c79e7c2ef17e8/src/nginxconfig/generators/conf/website.conf.js#L121